components/channel_groups_manage_modal/channel_groups_manage_modal.tsx

```diff
@@ -23,55 +24,54 @@ import Menu from 'components/widgets/menu/menu';
 
 import * as Utils from 'utils/utils.jsx';
 
-class ChannelGroupsManageModal extends React.PureComponent {
-    static propTypes = {
-        channel: PropTypes.object.isRequired,
-        intl: intlShape.isRequired,
-        actions: PropTypes.shape({
-            getGroupsAssociatedToChannel: PropTypes.func.isRequired,
-            unlinkGroupSyncable: PropTypes.func.isRequired,
-            patchGroupSyncable: PropTypes.func.isRequired,
-            getMyChannelMember: PropTypes.func.isRequired,
-            closeModal: PropTypes.func.isRequired,
-            openModal: PropTypes.func.isRequired,
-        }).isRequired,
+type Props = {
+    channel: any;
+    intl: any;
+    actions: {
+        getGroupsAssociatedToChannel: (channelId: string, searchTerm: string, pageNumber: number, DEFAULT_NUM_PER_PAGE: number) => any;
+        unlinkGroupSyncable: (itemId: string, channelId: string, type: any) => any;
+        patchGroupSyncable: (itemId: string, channelId: string, GroupsSyncableTypeChannel: any, params: {scheme_admin: boolean}) => any;
```

Comment:
Same as above, changing ```groupsSyncableTypeChannel``` to type ```SyncableType``` causes the below error: 

```
 Argument of type 'string' is not assignable to parameter of type 'SyncableType'.

67         this.props.actions.patchGroupSyncable(item.id, this.props.channel.id, Groups.SYNCABLE_TYPE_CHANNEL, {scheme_admin: isChannelAdmin}).then(async () => {
```

Changing it to type ```string``` also solved the error. 
As ```SYNCABLE_TYPE_CHANNEL``` in ```mattermost-redux/constants/groups``` is a string, I hope it would be okay to use type ```string``` instead of ```SyncableType```.

Thank you for the reminder about using lowerCamelCase, I'll remember to use it instead of CamelCase in the future :)